BCStudentSoftwareDevTeam/lsf
# Fewer form confirmation 381

This pr fixes issue #381 

Summary: 
  The previous LSF creation page had a series of confirmations that seemed redundant to users. 

- This PR has condensed the confirmations to two clicks, an "Add Student" button that will trigger a confirmation modal. 
- We have also changed the old table to a data table that will repopulate with all of the current users created and assigned forms. 
- Lastly, we changed the way cooked were being used on this page so that they will keep the fields populated after a form is created for easier batch adding, we also made sure the cookies from this page and the supervisorPortal work together. 

To Test: 
- Set the flask environment to staging, upon page load, you should see forms pre-loaded into the table below. You can then fill out a form for a student and see that after fully adding the form most of the fields will be populated for another add if you choose to do so. 
- If you go to the supervisor portal page and complete a search you should see that the cookies from both pages don't interfere with one another. 
- Run tests suite and make sure everything passes.
